### ISSUE TITLE
Librefox development team

### ISSUE THREAD

**Issue by intika** [intika/Librefox#72]
Librefox will probably need more people to work on it later on... 

So if you are interested in being a member of the future development team please reply here and indicate in what aspect of the browser you want to work. 

Current main aspect: 

- Building (Windows/Linux/Mac/Android) 
- Watching changes on new Firefox version (to patch changes that need to be patched) 
- Design (Icon, website, Github README)  
- Linguist (Checking the grammar of all the text around Librefox)  
- C++ developer (core settings)
- JavaScript developer (gui/extension/etc.)
- Communication (Advertising to make the project more known) 

Add an aspect if i forget one

**Comment by intika** [comment]
Current candidate @samuel8941

**Comment by intika** [comment]
Tagging Librefox related github users : @brainscar, @nhynes, @yoasif, @szepeviktor, @theel0ja,
@Eloston, @dimqua, @Atavic, @antuketot76, @maxidor, @anchev, @samuel8941, @shaynem, @elypter, @elypter and all the others i forget to mention.

**Comment by AnXh3L0** [comment]
Hi, I'd like to help with the design :+1:

**Comment by Atavic** [comment]
I can check grammar.

**Comment by shaynem** [comment]
I can assist with mac builds

**Comment by intika** [comment]
Thank you all for your answers; in the current situation things will just go through github PR open to anyone willing to help the next step would be a website i think @samuel8941 is preparing one in its fork... the site will be published just before the release... then we will see what happen either things will get more organized or it will remain as it is here :)

**Comment by colbycdev** [comment]
I'd like to work on design and/or communication

**Comment by dimqua** [comment]
I can translate into Russian.

**Comment by intika** [comment]
@atbanpc @Atavic @theel0ja @dimqua @colbycdev @AnXh3L0 
can you please send me what ever you use to chat (skype/xmpp/etc) to intika@librefox.org
Thanks

**Comment by dimqua** [comment]
@intika could I suggest to use [Gitter](https://gitter.im/) as a development chat? The reasons are:
* It's accessible by everyone who has GitHub/GitLab/Twitter account;
* It can be used trough IRC client, see [irc.gitter.im](https://irc.gitter.im/) for more info;
* It's bridged with a [Matrix](https://matrix.org/) network.

**Comment by intika** [comment]
@dimqua yeah why not if we need to have something like a general meeting or something sure we'll be using that but nothing on the project's main page because it would be an other channel to watch and answer for support.

**Comment by colbycdev** [comment]
I personally prefer telegram because:
-It's highly secure and private
-You can talk in a group, supergroup, or PM
-All you need is a phone #, and no password to remember
-Bots. Very useful, for instance @MissRose_bot
-You can have a private internal chat, and a public one for support 
Your choice, though. I'm on Gitter as well

**Comment by elypter** [comment]
requiring a phone number is not very private. even among those that require a phone number signal is a better choice

**Comment by colbycdev** [comment]
@elypter It's to keep out spammers and bots
And it allows most VOIP numbers 

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by intika** [comment]
Indeed in this case signal will be a better choice... but anyway we are not there yet ;)

**Comment by Atavic** [comment]
:+1: for @dimqua irc proposal.

**Comment by anchev** [comment]
> I personally prefer telegram because:
> -It's highly secure and private

On the contrary. Do some research.

> -All you need is a phone #, and no password to
> remember

That very statement contradicts the first one.

FWIW (and intika knows about that) XMPP + OMEMO is a
very secure and private FOSS combination. Also bear in
mind that security and privacy are also a matter which
relates directly to lower protection rings (down to
the hardware).

In any case this project doesn't seem to need ultimate
privacy as it is a public and open one. A self-hosted
mailing list and git repo would be fine IMO, rather
than depend on Microsoft owned platform.

**Comment by intika** [comment]
> @dimqua Gitter is old news now. Any chat I've seen hasn't been used in over six months
> 
> _Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

This is why such thing should not be promoted in the main page but it could be used for a meeting at a fixed date/time with everybody for instance

**Comment by colbycdev** [comment]
@intika Gotcha

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Techguyprivate** [comment]
What about splitting tor development and librefox ? 

What about having a dev or beta and stable releases? 

Is it possible?

**Comment by intika** [comment]
@Techguyprivate this is too soon for that we will see later on; dev/beta release is already in the todo list but this is not something important right now...

**Comment by shreyasminocha** [comment]
Can help with design, javascript, and language.

**Comment by intika** [comment]
Thank you Shreyas, adding you to the team ;)

**Comment by shreyasminocha** [comment]
@intika Thanks.
PS: You got my name right the first time 😆.

**Comment by atbanpc** [comment]
I noticed that Linux building is empty. I can help out with that as well.
